router/middleware/config.go

```diff
@@ -39,6 +39,7 @@ func setupConfig(c *cli.Context) *model.Settings {
 		Secret: c.String("agent-secret"),
 		Admins: sliceToMap2(c.StringSlice("admin")),
 		Orgs:   sliceToMap2(c.StringSlice("orgs")),
+		OwnersWhitelist: sliceToMap2(c.StringSlice("repo-owner")),
```

Comment:
Does this work? `repo-owners` is specified at https://github.com/laszlocph/woodpecker/pull/119/files#diff-a1e14c06700b04b423b787fd33d25b34R107 
Mind the plural form